practicals/LLM_prac.ipynb

```diff
@@ -0,0 +1,411 @@
+{
```

Comment:
<p>Maybe add a note mentioning that we def. want to cover this whole section during the practical, right? While section 2 and 3 might contain a lot of code/content so during the practical itself we might want to skip certain parts of it or focus on some specific sections</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2023/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>